Snabble/UI/Payment/CustomerCardCheckoutViewController.swift

```diff
@@ -50,10 +51,23 @@ final class CustomerCardCheckoutViewController: UIViewController {
     override public func loadView() {
         let contentView = UIView(frame: UIScreen.main.bounds)
         contentView.backgroundColor = .systemBackground
+        contentView.restrictDynamicTypeSize(from: nil, to: .extraExtraExtraLarge)
+
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.backgroundColor = .systemBackground
+        scrollView.showsVerticalScrollIndicator = false
+        scrollView.alwaysBounceVertical = false
+
+        let contentLayoutGuide = scrollView.contentLayoutGuide
+        let frameLayoutGuide = scrollView.frameLayoutGuide
+
+        let wrapperView = UIView()
+        wrapperView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Cause of the `contentLayoutGuide` and `frameLayoutGuide` a wrapper view isn't needed anymore. Use the `contentLayoutGuide` as edges for all content views and layout the views as usual it should work.